remindmoi_bot_handler.py

```diff
@@ -41,6 +48,9 @@
 `add 1 day clean the dishes`
 `add 10 hours eat`
 
+To add a event to your calendar the format is:
+`--calendar 16-06-2020 18:00` (dd/mm/yyyy) (hour in utc time)
```

Comment:
Match the format with the example (dd-mm-yyyy) also, this needs to be timezone aware, not just utc

Reply:
but the timezone will be another input in the string to create the event because we don't have a datetime obj, is just a date calculated from a string.